packages/block-library/src/post-comments-form/edit.js

```diff
@@ -43,6 +50,9 @@ export default function PostCommentsFormEdit( {
 			</BlockControls>
 			<div { ...blockProps }>
 				<CommentsForm postId={ postId } postType={ postType } />
+				<VisuallyHidden id={ instanceIdDesc }>
+					{ __( 'Comments form disabled on back-end.' ) }
+				</VisuallyHidden>
```

Comment:
This was helpful to explain why and gets announced well in my testing. Instead of "back-end," using "editor" may be more consistent.

"Comments form is disabled within the editor."

Reply:
Done.